RELEASE-NOTES.md

```diff
@@ -1,4 +1,60 @@
-# PatternFly Albatross Milestone (2019-07-02)
+## 2019.04 release notes (2019-07-24)
+- @patternfly/patternfly@2.23.0
+
+**About modal:** Updated break-word on content area instead of break-all ([#2035](https://github.com/patternfly/patternfly-next/pull/2035))
+
+**App launcher**
+* Added button example to menu item ([#2006](https://github.com/patternfly/patternfly-next/pull/2006)) 
+* Added support for right and top alignment ([#2081](https://github.com/patternfly/patternfly-next/pull/2081))
+
+**Charts**
+- Adjusted chart vars for react-charts ([#2020](https://github.com/patternfly/patternfly-next/pull/2020), [#2024](https://github.com/patternfly/patternfly-next/pull/2024))
+- Adjusted 3 chart tooltip color vars and adds two new ones. This will ensure tooltips can be seen over the current background color. ([#2038](https://github.com/patternfly/patternfly-next/pull/2038))    
+- Added individual padding vars for donut charts ([#2068](https://github.com/patternfly/patternfly-next/pull/2068))
+
+**Chip:** updated padding on chip label ([#2063](https://github.com/patternfly/patternfly-next/pull/2063))
```

Comment:
in the previous notes, a single item under a component is it's own list with 1 item. Just want to make sure doing it this way is on purpose?